services-directions/src/test/java/com/mapbox/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -839,6 +839,30 @@ public void testWithInterceptor() throws Exception {
     assertEquals(interceptor, mapboxDirections.interceptor());
   }
 
+  @Test
+  public void testWithNetworkInterceptor() throws Exception {
```

Comment:
Minor thing Could we rename the test so it's clearer (although not consistently applied 👀 at the "naming" convention of the rest of the tests)? We should also remove `test` prefix from tests names, it's implicit with the usage of `@Test` annotation.

Reply:
oh yeah completely agree! i was only being consistent within this change. addressing naming conventions can be part of a separate PR to maintain consistency